Documentation/ApiOverview/SiteHandling/SiteSets.rst

```diff
@@ -0,0 +1,345 @@
+
+    ..  group-tab:: Composer-based installation
+
+        ..  code-block:: bash
+
+            vendor/bin/typo3 site:sets:list
+
+    ..  group-tab:: Legacy installation
+
+        ..  code-block:: bash
+
+            typo3/sysext/core/bin/typo3 site:sets:list
+
+
+..  _site-sets-example-site-package:
+
+Example: Using a set within a site package
+==========================================
+
+You can see an example of using a sets within a site package in the extension
+:t3ext:`site-package`,
+`t3docs/site-package (Source on GitHub) <https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code>`__.
+
+The site package example extension has the following file structure:
+
+..  directory-tree::
+    :show-file-icons: true
+
+    *   Configuration
+
+        *   Sets
+
+            *   SitePackage
+
+                *   config.yaml
+                *   constants.typoscript
+                *   page.tsconfig
+                *   settings.yaml
+                *   setup.typoscript
+
+            *   ...
+
+    *   Resources
+
+        *   ...
+
+    *   composer.json
+    *   ...
+
+..  _site-sets-example-site-package-set:
+
+Defining the site set with a fluid_styled_content dependency
+------------------------------------------------------------
+
+As our example site package only contains one site set the name of that set
+is the same like the composer name of the site package.
+
+The site package depends on EXT:fluid_styled_content. Therefore the two
+sets provided by that system extension are included as dependencies:
+
+..  include:: _Sets/_site-package/_config.rst.txt
+
+If you need additional dependencies, you can find all available sets with the
+console command :ref:`bin/typo3 site:sets:list <site-sets-cli>`.
+
+..  _site-sets-example-usage:
+
+Using the site set as dependency of a site
+------------------------------------------
+
+After the example site package is installed, you can include the site set
+in your site configuration:
+
+..  literalinclude:: _Sets/_site-package/_site_config.yaml
+    :language: yaml
+    :caption: config/sites/<some_site>/config.yaml | typo3conf/sites/<some_site>/config.yaml
+
+..  _site-sets-example-typoscript:
+
+Loading TypoScript via the site package's set
+---------------------------------------------
+
+The example site package also loads its TypoScript by placing the files
+:file:`constants.typoscript` and :file:`setup.typoscript` into the folder of the
+site set. These TypoScript files use :typoscript:`@import` statements to import
+files from the extension`s directory :path:`Configuration/TypoScript`:
+
+..  include:: _Sets/_site-package/_setup.rst.txt
+
+Dependant TypoScript is included by including dependant sets and not by
+TypoScript imports.
+
+..  _site-sets-example-settings:
+
+Using the site set to override default settings
+-----------------------------------------------
+
+You can use the file
+:file:`EXT:site_package/Configuration/Sets/SitePackage/settings.yaml` to
```

Comment:
Maybe this could mislead people to think that EXT:site_package is an extension they get from TYPO3. We could maybe phrase "You can use the file ... of your custom sitepackage extension [link to sitepackage docs] ..."?